mysql-test/suite/perfschema/r/socket_instances_func.result

```diff
@@ -80,24 +80,26 @@ Expect 1
 1
 # Characteristics of 'server_tcpip_socket' entry
 #    Server listening socket, TCP/IP
-#    There is only one entry with 'wait/io/socket/sql/server_tcpip_socket'.
-#    It shares the same thread id as 'wait/io/socket/sql/server_unix_socket'.
-SELECT COUNT(*) = 1 AS 'Expect 1'
+#    There are two entries with 'wait/io/socket/sql/server_tcpip_socket',
+#    for [::] and for 0.0.0.0.
+#    They share the same thread id with 'wait/io/socket/sql/server_unix_socket'.
+SELECT COUNT(*) = 2 AS 'Expect 1'
 FROM performance_schema.socket_instances
 WHERE EVENT_NAME = 'wait/io/socket/sql/server_tcpip_socket';
 Expect 1
 1
 # Get the 'server_tcpip_socket' thread id
-SELECT THREAD_ID INTO @thread_id
+SELECT DISTINCT(THREAD_ID) INTO @thread_id
 FROM performance_schema.socket_instances
 WHERE EVENT_NAME = 'wait/io/socket/sql/server_tcpip_socket';
 #    Check the content.
-SELECT THREAD_ID = @thread_id
+SELECT COUNT(*) = 2 AS 'Expect 1'
+FROM performance_schema.socket_instances
+WHERE EVENT_NAME = 'wait/io/socket/sql/server_tcpip_socket'
+      AND THREAD_ID = @thread_id
 AND (IP = '0.0.0.0' OR IP = '::')
 AND PORT = @port
-AND STATE = 'ACTIVE' AS 'Expect 1'
-FROM performance_schema.socket_instances
-WHERE EVENT_NAME = 'wait/io/socket/sql/server_tcpip_socket';
+AND STATE = 'ACTIVE';
 Expect 1
```

Comment:
Reverted to the original pre-patched state this query.
The query was fine as it was, but I tried to make it clear in the query that two rows are expected. Could not find a way to do that, so just reverted the change. The only difference now is in the result file: two rows are returned instead of one.